src/components/SwitchToggle.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import styled from 'styled-components/macro';
+import PropTypes from 'prop-types';
+
+const Checkbox = styled.input.attrs({ type: 'checkbox' })`
+  border: 0;
+  clip: rect(0 0 0 0);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+`;
+
+const Label = styled.label`
+  align-items: center;
+  display: flex;
+  min-height: 2rem;
+  padding-left: 3.5rem;
+  position: relative;
+  user-select: none;
+  &::before {
+    background-color: ${theme => theme.input};
+    border: 1px solid
+      ${({ checked, theme }) => (checked ? theme.border : '#ece9f1')};
```

Comment:
Oops, it seems you missed this one: `checked ? theme.borderActive : theme.border` 😄